Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG ruby_version=2.7.6
+ARG ruby_version=3.2.0
```

Comment:
I was asked to change this to 3.2 - as the Docker image patch versions are updated frequently. Might be worth checking with platform reliability, but 3.2 should be ok either way.

Reply:
thanks @gclssvglx. I've changed it to 3.2 for Docker.